api/stats.js

```diff
@@ -46,17 +49,24 @@ export class Stats {
             collectedDays.add(key);
 
             let fields = message.embeds[0].fields;
-            let firstColumn = fields[0].value.split('\n');
-            let secondColumn = fields[2].value.split('\n');
+            let rows = fields[0].value.split('\n');
             let i = 0;
-            while (i < firstColumn.length) {
-                let [username, posts] = firstColumn[i].split(' ');
-                let [words, letters] = secondColumn[i].split(' ');
+            for (let row of rows) {
+                // "Winner Crespo (5 | 50 | 2500)"
+                let [username, values] = row[i].split('(');
+                if (!username || !values) {
+                    // we couldn't parse
+                    break;
+                }
+                // clean input
+                username = username.trim();
+                values = values.replaceAll(' ', '').replaceAll(')', '');
+
+                let [posts, words, letters] = values.split('|');
                 stats[username] = stats[username] || { posts: 0, words: 0, letters: 0 };
-                stats[username].posts += Number(removeParentheses(posts));
+                stats[username].posts += Number(posts);
                 stats[username].words += Number(words);
-                stats[username].letters += Number(removeParentheses(letters));
-                i++;
+                stats[username].letters += Number(letters);
```

Comment:
This is basically going from 2 columns to one

Before
<img width="346" alt="image" src="https://user-images.githubusercontent.com/1899538/197684101-7cbc1761-6049-4980-acaf-9b9f378a0197.png">

After
<img width="335" alt="image" src="https://user-images.githubusercontent.com/1899538/197684076-b44610ca-1aa5-4649-913c-9a176f12e60b.png">